### DISCUSSION TITLE
Texte du bouton Enregistrer paramétrable

### DISCUSSION THREAD

**Discussion by drogu02** [MTESSDev/FRW#24] [Annonces]
Bonjour, j'aimerais pouvoir changer le texte du bouton Enregistrer. Donc paramétrable.

Par exemple, Sauvegarder et reprendre plus tard.

**Comment by j0ber** [comment]
Bonjour,

Nous ne voulons pas permettre la modification du texte de ce bouton afin que tous les formulaires soient identiques. Ça fait partie du coeur de la solution. Si beaucoup d'autres utilisateurs trouvent que le terme "Enregistrer" ne convient pas, on conviendra du texte à utiliser, mais il ne sera pas modifiable, donc identique pour tous.

De plus, si tu affiches le message incitatif d'enregistrement, l'utilisateur a toute l'info lui indiquant :  
![image](https://user-images.githubusercontent.com/26974817/222748026-f692f0ca-2817-4a77-9c90-1b7b66ceb4dd.png)

Aussi la fenêtre modale affichée sur le clic du bouton fourni également cette information: 
![image](https://user-images.githubusercontent.com/26974817/222756461-2f61faaa-bfab-45e9-bb2b-b51156f48447.png)

**Comment by drogu02** [comment]
Parfait je vais utiliser l'avis!  Merci!
